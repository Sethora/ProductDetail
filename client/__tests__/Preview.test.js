import Preview from '../src/containers/preview';

describe('Preview component', () => {
  test('should render the component', () => {
    const wrapper = shallow(<Preview product={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});