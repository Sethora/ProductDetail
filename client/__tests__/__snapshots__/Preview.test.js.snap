// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview component should render the component 1`] = `
<styled.div>
  <ImgInner
    photos={Array []}
    selected={0}
  />
  <Carousel
    click={[Function]}
    photos={Array []}
  />
</styled.div>
`;
